Core/include/Acts/EventData/detail/fittable_type_generator.hpp

```diff
@@ -107,14 +109,16 @@ constexpr auto type_generator() {
 /**
  * Type alias which unpacks the hana tuple generated in `type_generator`
  * into an `std::variant`.
+ * @tparam parameter_indices_t Parameter indices enum
  * @tparam meas_meta Factory meta function for measurements.
- * @tparam W max number of parameters.
  */
-template <template <ParID_t...> class meas_meta, size_t W>
-using type_generator_t = typename decltype(hana::unpack(
-    type_generator<meas_meta, W>(), hana::template_<std::variant>))::type;
+template <typename parameter_indices_t,
+          template <parameter_indices_t...> class meas_meta>
+using type_generator_t = typename decltype(
+    hana::unpack(type_generator<parameter_indices_t, meas_meta>(),
+                 hana::template_<std::variant>))::type;
 
 /// @endcond
 }  // namespace detail
 /// @endcond
-}  // namespace Acts
+}  // namespace Acts
```

Comment:
Nitpick: could you add the missing newlines?

I finally found out why Github complains about the missing newline: the POSIX standard requires that all lines end with a newline [2]; see e.g. [1] for further explanations.

[1]: https://thoughtbot.com/blog/no-newline-at-end-of-file
[2:] https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206

Reply:
I received once a comments on a MR that it should be removed. Since then I did it in all my MRs/PRs. Unfortunately I'm not sure who commented it but maybe the person reads this comment by chance^^

Reply:
Might have been to avoid non-necessary changes; I do not know. The standard seems to be quite clear: every line, including the last one, has to end with a newline.

Reply:
This is very much varying. Let's just keep changing / removing newlines at the end of files out of changesets? Just don't commit them and they'll stay the same.

Reply:
Basically, it doesn't matter if they are there or not there, just don't change it in any file.

Reply:
I read a bit more about this. It seems that since C++11 it does not matter anymore for the compiler. That does not mean it is correct. The fact that e.g. git/Github explicitly marks this should be an indication.

So, I think we should not removing newlines that are already there. We can hold of adding missing newlines to reduce the changesets.

Reply:
Github only marks that there is a change. That makes sense. You can tell git to ignore newlines I believe. Let's try not to get hung up on this absolutely minimal detail. I think we should just ignore this as much as possible. In the end, nobody cares.

Reply:
I changed all cases (if I didn't miss anything) to what it currently is